src/ZendModuleProvider.php

```diff
@@ -177,7 +179,7 @@ public function getHydratorConfig() : array
         return $this->convert($this->module->getHydratorConfig());
     }
 
-    public function getInputFilterConfig()
+    public function getInputFilterConfig() /* : array */
```

Comment:
does it help with anything now? Or it is just added because it was missed in that place as 'hard' RTH?

Reply:
I obviously forgot to add this in the first release.
Therefore I thought adding that comment to remind me that I have to add it in the next major release (as its a bc in case of extending the `ConfigProvider`. Should have put a final on it since it should be never extended at all...)